validator/webui/index.html

```diff
@@ -389,6 +391,7 @@
 
           // The status bar below the editor.
           var statusBar = this.$.statusBar;
+          statusBar.status = 'PASS';  // Start out valid.
```

Comment:
True it is. What do you think of running the validator on this text once the page loads? It looks to me like we're setting the validation to pass without running the validator.